src/GerdaFitter.cc

```diff
@@ -259,8 +260,21 @@ GerdaFitter::GerdaFitter(json outconfig) : config(outconfig) {
 
         // loop over requested histograms in file
         for (auto& elh : el.value().items()) {
+            // get rebin factor before reading all histograms
+            auto rebin_x = elh.value().value("rebin-factor-x", 1) * elh.value().value("rebin-factor", 1);
```

Comment:
I would use `"rebin-factor"` as a factor to apply to all histogram dimensions, like `"rebin-factor-xyz"`...

Reply:
Not only the x

Reply:
If you want to have different rebinnigs in x and y how would you do that?